src/components/AdminDashboard/AntTableComponents/AntTable.js

```diff
@@ -111,8 +94,8 @@ function AntTable(props) {
           onChange: onSelect,
         }}
         pagination={{
-          position: ['topRight', 'bottomCenter'],
-          total: listToUse ? listToUse.length : 0,
+          position: ['bottomCenter'],
+          total: incidents ? incidents.length : 0,
           showTotal(total, range) {
             return `${range[0]}-${range[1]} of ${total} items`;
```

Comment:
From my understanding, this is a custom Ant Table to be utilized throughout the App. Are we sure none of those props that are being deleted are being utilized whereever this table is being utilized? Has functionality been tested where this table is being used to make sure no bugs are created? Just want to make sure with these types of changes.

Reply:
I did a search for usages of AntTable, and there are only 2: Incidents.js (which has been cleaned up and simplified) and ApprovedIncidents.js (which can/will be deleted, because Incidents.js can be used now to display any list of incidents you hand it.)

I'll admit I didn't spend a lot of time on AntTable.  But the biggest changes here are the removal of unused props/imports, and the removal of listToUse, which is no longer necessary because it simply renders the incidents you pass into it.